pymc_marketing/clv/models/basic.py

```diff
@@ -165,55 +163,13 @@ def _fit_DEMZ(self, **kwargs) -> az.InferenceData:
             return pm.sample(step=pm.DEMetropolisZ(), **sampler_config)
 
     @classmethod
-    def load(cls, fname: str):
```

Comment:
https://github.com/pymc-labs/pymc-marketing/issues/1356 should be considered for the `load` method replacing this.